docs/current/guides/421437-work-with-host-filesystem.md

```diff
@@ -135,24 +135,24 @@ The following example shows how to obtain a reference to the host working direct
 </TabItem>
 </Tabs>
 
-The exclusion pattern overrides the inclusion pattern, but not vice-versa. The following example demonstrates by obtaining a reference to the host working directory containing all files except `*.rar` files:
```

Comment:
I would prefer not to delete the existing snippets. Can we restore this snippet to its original form and place the new glob pattern example as a new example in the same section, after this one?